Bio/Data/CodonTable.py

```diff
@@ -62,8 +57,8 @@ class CodonTable:
     # Not always called from derived classes!
     def __init__(
         self,
-        nucleotide_alphabet=nucleotide_alphabet,
-        protein_alphabet=protein_alphabet,
+        nucleotide_alphabet=None,
```

Comment:
How would you feel about changing the argument and attribute names from ``nucleotide_alphabet`` and ``protein_alphabet`` (which were alphabet objects) to something like ``nucleotide_letter`` and `protein_letters`` (which are now presumably expected to be strings or similar)? i.e. Make the API change more explicit?

Reply:
Using "alphabet" for a string of letters is consistent with the attribute names used in e.g. `Bio.motifs`, `Bio.Align.PairwiseAligner`, and `Bio.Align.substitution_matrices`.

Reply:
I suppose I should have made this argument more strongly back then. Oh well.